Lib/test/_test_multiprocessing.py

```diff
@@ -5010,12 +5010,21 @@ def create_and_register_resource(rtype):
                 _resource_unlink(name1, rtype)
                 p.terminate()
                 p.wait()
-                time.sleep(2.0)
-                with self.assertRaises(OSError) as ctx:
-                    _resource_unlink(name2, rtype)
-                # docs say it should be ENOENT, but OSX seems to give EINVAL
-                self.assertIn(
-                    ctx.exception.errno, (errno.ENOENT, errno.EINVAL))
+
+                deadline = time.monotonic() + 60
+                while time.monotonic() < deadline:
+                    time.sleep(2.0)
```

Comment:
Would it be possible to use a shorter sleep to make the test faster? Like sleep(.5).